Lib/json/tool.py

```diff
@@ -68,12 +72,42 @@ def main():
             outfile = open(options.outfile, 'w', encoding='utf-8')
         with outfile:
             for obj in objs:
-                json.dump(obj, outfile, **dump_args)
+                if with_colors:
+                    json_str = json.dumps(obj, **dump_args)
+                    outfile.write(colorize_json(json_str))
+                else:
+                    json.dump(obj, outfile, **dump_args)
                 outfile.write('\n')
     except ValueError as e:
         raise SystemExit(e)
 
 
+color_pattern = re.compile(r'''
+    (?P<string>"(\\"|[^"])*?") |   # String
+    (?P<number>[\d\-+.Ee]+)    |   # Number
+    (?P<boolean>true|false)    |   # Boolean
```

Comment:
`true`/`false` are not valid Python syntax, why not just:

```suggestion
    (?P<boolean>True|False)    |   # Boolean
```

Reply:
Because we're handling JSON output, not Python input.